### PR TITLE
Extending order

### PR DESCRIPTION
**Pull request by nastiatikk**
Status: **Ready for Review**
Reviewers: @avelinet @jeffkamo @kpeatt @ry5n @mlworks @cole-sanderson 

## Changes
A minor tweak in documentation about:
* where `%extend` should go in the selectors order
* how placeholder class should be named

## Todos:
* [x] Tweak grammatical mistakes
* [x] Designer +1
* [x] Designer +1

## Notes
* Inform designers and engineers as this is often wrong used

### CODE REVIEW

File: css/sass-best-practices/readme.md [hunk 1]
```diff
@@ -123,6 +123,47 @@ Also notice how we still included the original `.c-button` class in the SCSS. Af
...
+    font-family: sans-serif;
+    text-transform: uppercase;
+}
+```
+
+If modifier is extended use the following name convention:
+
+```
+.c-component.c--modifier,
+%c-component--modifier {
```
Comment (kpeatt) [inline css/sass-best-practices/readme.md]: When would you do this>?

Comment (nastiatikk) [inline css/sass-best-practices/readme.md]: It happens when we need to extend our styles on the ajaxed content. 
```
.c-button.c--secondary,
%c-button--secondary {
    ...
}
```

Comment (kpeatt) [inline css/sass-best-practices/readme.md]: Nod, that makes sense. I hadn't thought about buttons.

**Comment by jeffkamo** [comment]
I just did a tweak of the English. This looks pretty good to me! +1

**Comment by jeffkamo** [comment]
I removed the "Inform designers/engineers" todo and made it a note. The reason is, because that task isn't a required todo for the purpose of completing and merging this PR.

File: css/sass-best-practices/readme.md [hunk 2]
```diff
@@ -123,6 +123,46 @@ Also notice how we still included the original `.c-button` class in the SCSS. Af
 
 This technique is described in detail in Chris Lamb's article [Mastering Sass Extends and Placeholders](http://8gramgorilla.com/mastering-sass-extends-and-placeholders/).
 
+### Declaration Order And Placeholder Class Names
+
+The placeholder class always goes after the regular selectors. In other works, make the placeholder class the last selector in a chain of comma separated selectors.
+
```
Comment (ry5n) [inline css/sass-best-practices/readme.md]: “In other works” → “In other words”?

Comment (jeffkamo) [inline css/sass-best-practices/readme.md]: Oh geez, that's my fault. Will fix.

**Pull request merged**